packages/invoices/src/OutboundShipment/DetailView/SidePanel.tsx

```diff
@@ -44,31 +39,29 @@ export const SidePanel: FC<SidePanelProps> = ({ draft }) => {
         />
       }
     >
-      <DetailPanelSection titleKey="heading.comment">
-        <TextArea
-          onChange={e => draft.update?.('comment', e.target.value)}
-          value={draft.comment}
-        />
-      </DetailPanelSection>
       <DetailPanelSection titleKey="heading.additional-info">
-        <Grid container key="additional-info">
+        <Grid container gap={0.5} key="additional-info">
+          <PanelRow>
+            <PanelLabel>{t('label.entered-by')}</PanelLabel>
```

Comment:
I thought that maybe the labels should be bold - but then checked zeplin and they aren't, but the values are lighter `#8f90a6` and right-aligned. I think they look better that way, more of a distinction between label and value:

<img width="343" alt="Screen Shot 2021-11-08 at 9 59 00 AM" src="https://user-images.githubusercontent.com/9192912/140661697-ba933856-bbc0-4376-9848-ee8434059fc8.png">

Reply:
Oh, nice, thanks. I will update in the next PR as I right align there!